Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -88,8 +73,8 @@ class TopTabCell: UICollectionViewCell, NotificationThemeable, TabTrayCell, Reus
         closeButton.isHidden = hideCloseButton
 
         favicon.image = UIImage(named: ImageIdentifiers.defaultFavicon)?.withRenderingMode(.alwaysTemplate)
-        favicon.tintColor = UIColor.theme.tabTray.faviconTint
         favicon.backgroundColor = .clear
+        favicon.tintColor = UIColor.theme.tabTray.faviconTint
```

Comment:
`UIColor.theme` is part of the legacy theme. We should use colors from `Theme`. I think I'll open a PR to rename this static var to legacyTheme to make this easier to see